src/app/client/src/app/modules/learn/components/learn-page/learn-page.component.ts

```diff
@@ -210,23 +210,11 @@ export class LearnPageComponent implements OnInit, OnDestroy, AfterViewInit {
     this.dataDrivenFilterEvent.emit(defaultFilters);
   }
   private getFrameWork() {
-    const framework = this.cacheService.get('framework' + 'search');
+    const framework = this.frameworkService.getDefaultCourseFramework();
     if (framework) {
-      return of(framework);
+      return framework;
     } else {
-      const formServiceInputParams = {
-        formType: 'framework',
-        formAction: 'search',
-        contentType: 'framework-code',
-      };
-      return this.formService.getFormConfig(formServiceInputParams, this.hashTagId)
-        .pipe(map((data: ServerResponse) => {
-          const frameWork = _.find(data, 'framework').framework;
-          this.cacheService.set('framework' + 'search', frameWork, { maxAge: this.browserCacheTtlService.browserCacheTtl });
-          return frameWork;
-        }), catchError((error) => {
-          return of(false);
-        }));
+      return of(false);
```

Comment:
return proper error object and handle it

Reply:
it was handled like this only, id form config was giving error.